.drone.yml

```diff
@@ -2154,7 +2153,9 @@ steps:
       # copy release archives to build directory
       - mkdir -p /go/artifacts/windows
       - find . -maxdepth 1 -iname "teleport*.zip" -print -exec cp {} /go/artifacts \;
-      # generate checksums
+      # make a copy of the Windows binaries with an Enterprise name (this is a Houston requirement)
```

Comment:
explain why this is needed.
something like "Windows tsh binary for enterprise setups is identical to OSS setups."

Reply:
Done!